crates/bitwarden-exporters/src/cxf/login.rs

```diff
@@ -65,10 +98,25 @@ impl From<Login> for BasicAuthCredential {
 
 impl From<Login> for CredentialScope {
     fn from(login: Login) -> Self {
-        CredentialScope {
-            urls: login.login_uris.into_iter().filter_map(|u| u.uri).collect(),
-            android_apps: vec![],
-        }
+        let (android_uris, urls): (Vec<_>, Vec<_>) = login
+            .login_uris
+            .into_iter()
+            .filter_map(|u| u.uri)
+            .partition(|uri| uri.starts_with(&format!("{}://", ANDROID_APP_SCHEME)));
```

Comment:
Thought: Every single time we use `ANDROID_APP_SCHEME` we append `://` to it, I wonder if we should just update `ANDROID_APP_SCHEME` to be `androidapp://`, which would avoid some allocations in this function.